lib/Utils/Log.php

```diff
@@ -54,6 +118,13 @@ protected static function echoDebug($data, $postpone)
         }
     }
 
+    /**
+    * Returns a filename and line number by performing a debug_backtrace.
+    *
+    * @since 0.1.0
+    *
+    * @param number $depth  depth to return.
```

Comment:
not sure of this makes a difference, but should it be `number` / `int` / `integer`?